mungegithub/mungers/approvers/owners.go

```diff
@@ -214,20 +229,51 @@ func IntersectSetsCase(one, other sets.String) sets.String {
 	return intersection
 }
 
+func NewApprovers(owners Owners) Approvers {
+	return Approvers{
+		owners:    owners,
+		approvers: map[string]Approval{},
+	}
+}
+
+func (ap *Approvers) AddApprover(name, reason, reference string) {
```

Comment:
I'm surprised golint isn't complaining about missing comments.  Can you explain the functionality of all the exported funcs